pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1763,6 +1763,15 @@ type BackupScheduleSpec struct {
 	// if MaxBackups and MaxReservedTime are set at the same time, MaxReservedTime is preferred
 	// and MaxBackups is ignored.
 	MaxBackups *int32 `json:"maxBackups,omitempty"`
+	// MaxCompletedBackups specifies the number of completed backups to retain. It should be greater than 0, defaults to 3.
+	// +kubebuilder:default:=3
+	// +kubebuilder:validation:Minimum:=1
```

Comment:
the old tc can not be applied anymore?

Reply:
The old `backupschedule` object will use the default value.